# apresentação

- pro slide do pangenoma, n vai ter gatilho perfeito, eu só cito de pangenomica no ultima slide e entro falando sobre:
	1. conceito de pangenoma (rapido)
	2. estudo com vacina
	3. sistemas complexos e openess







# pangs de sc

1- gang li: ele usou machile learning pra separar o CAR - sendo que o car dele é todo esquisito
2- song: fizeram pipeline que vai desde montagem até anotação

os mais promissores são: gang li, peter e Mccarthy-Fitzpratick

gang li: homolog groups com machine learning
peter: map to pan
mf: 


#Mccarthy-Fitzpratick

1- Many tools for pan-genome analysis have been
published in recent years, which utilize methods such as
whole-genome alignment, read mapping, clustering algo-
rithms or de Bruijn graph construction [12–16].

2- Although the concept of the species pan-genome is well-
established in comparative prokaryote genomics, it has only
recently been extended to comparative intraspecific studies
of eukaryotes. This is despite repeated observation of intra-
specific genomic content variation in eukaryotes dating
back to the first intraspecific comparative analyses of Sac-
charomyces cerevisiae genomes in the mid-2000s

3- We have found strong evidence for pan-genomic
structure within all four fungal species. In line with previ-
ous analyses of other eukaryotes, we found that approxi-
mately 80–90 % of fungal species’ pan-genomes are
composed of core genes, while the remainder is composed
of strain or lineage-specific accessory genes. Analysis of
the origin of fungal pan-genomes suggests that fungal
accessory genomes are enriched for genes of eukaryotic
origin and arise via eukaryotic innovations such as gene
duplication as opposed to large-scale HGT. Functionally,
fungal core genomes are enriched for both housekeeping
processes and essential survival processes in pathogenic
species, whereas many fungal accessory gene models are
found within clusters in the terminal and subterminal
regions of genomes and are enriched for processes that
may be implicated in fungal pathogenicity or antimicro-
bial resistance	

# guy franck

(openess)
1-Among sequenced bacterial species, some exhibit a closed pangenome, for
example Staphylococcus aureus (α ¼ 1.84), Streptococcus pyogenes (α ¼ 1.88),
Ureaplasma urealyticum (α ¼ 2.5) or the extreme case of Bacillus anthracis
(α ¼ 5.6). Others display an open pangenome, like Bacillus cereus (α ¼ 0.65) or
the cyanobacteria Prochlorococcus marinus (α ¼ 0.80). Note than when α is equal
or very close to 1, the pangenome is still open, but the rate of acquisition of new
genes is very slow. This is the case of Escherichia coli (α ¼ 1.04), Streptococcus
agalactiae (α ¼ 1.05), or Streptococcus pneumoniae (α ¼ 0.98) (Tettelin et al.
2008).


2- Using
de novo assembly and a specific detection pipeline, it could be determined that the
yeast core-genome contained 4940 Open Reading Frames (ORFs) whereas 2856
ORFs were variable within the population, for a total of 7796 ORFs constituting the
pangenome (Peter et al. 2018) (Table 1). Core ORFs were mostly found in one copy
per haploid genome, while ca. 20% of variable ORFs were absent or present in more
than one copy. The authors subsequently looked at the origin of these variable ORFs
and classified them in three different groups, based on their phylogeny: ORFs with
their closest ortholog in another S. cerevisiae strain and consistent with genome
phylogeny were considered as being ancestral acquisitions; ORFs with their best
ortholog in another Saccharomyces species were considered to be introgressions; and
finally ORFs more related to another yeast species outside the Saccharomyces
complex were treated as horizontal gene transfers (HGT)

- [motivos de eucarioto ser mais dificil] Calculation of eukaryotic core- and pangenomes is significantly more complex
for several reasons: (1) the abundance of transposable elements, including novel
undescribed transposons absent from dedicated databases; (2) the morcellated nature
of genes, particularly in young eukaryotes; (3) the presence of large gene families
that make orthologue identification tedious; and (4) the relative incompleteness of
genomic sequences, particularly of those containing numerous repeats

#Hu

(
1-







#paragrafo de pangenoma:

1- advento da big data na biologia molecular, 
	falar da quantidade de sequenciamentos no GOLD e eucariotos, como a sc	
	[imagem antes e depois da bigdata]
?- aqui poderia entrar a motivação pra pangenoma de humanos, bgl da vacina
2- definição certinha de pangenoma

TIPOS DE IMPLEMENTAÇÃO

3- CAR
4- graph bases pangenomes

#objetivos:

- 

#métodos




desafios da minha escrita:
- COMO DEFINIR CERTINHO UM PANGENOMA MEU DEUS
- 
